peer/network.go

```diff
@@ -327,7 +333,7 @@ func (n *network) AppRequest(ctx context.Context, nodeID ids.NodeID, requestID u
 	var req message.Request
 	if _, err := n.codec.Unmarshal(request, &req); err != nil {
 		log.Debug("forwarding AppRequest to SDK router", "nodeID", nodeID, "requestID", requestID, "requestLen", len(request), "err", err)
```

Comment:
We use the SDK's networking layer here which is just the router.... so I'm indifferent on whether we should update this log or not but will defer to others' opinions.